clang/lib/Headers/arm_acle.h

```diff
@@ -32,7 +32,7 @@ extern "C" {
 #define __isb(i) __builtin_arm_isb(i)
 #endif
 
-/* 8.4 Hints */
+/* 7.4 Hints */
```

Comment:
If we're matching the section titles then they should have the extra `.` on the end of the number, so here `7.4. Hints`. Just to make searching more obvious.

Though in Firefox and Chrome they struggled to find it but that's down to the HTML formatting I think. At least visually it'll match up.

Reply:
HTML version has the extra `.` https://arm-software.github.io/acle/main/acle.html#hints but the PDF one does not https://github.com/ARM-software/acle/releases/download/r2023Q2/acle-2023Q2.pdf - the solution, probably, depends on what is linked as suggested in a comment above: HTML or PDF version.